src/mixin/describable-entity-mixin.js

```diff
@@ -0,0 +1,27 @@
+import { performSirenAction } from '../es6/SirenAction.js';
+
+export const DescribableEntityMixin = superclass => class extends superclass {
```

Comment:
I'd like to hear more about what lead to this pattern. 

The plan is to avoid specific mixins but plans sometimes don't work in the real world.

Reply:
Oh, I see how they are being used. Interesting. 

I'd want to separate them from the mixins that are framework specific.